src/main/java/by/exadel/internship/controller/SchedulingController.java

```diff
@@ -0,0 +1,48 @@
+package by.exadel.internship.controller;
+
+
+import by.exadel.internship.annotation.AdminAccessControl;
+import by.exadel.internship.dto.time_for_call.UserTimeSlotWithUserIdDTO;
+import by.exadel.internship.dto.UserDTO;
+import by.exadel.internship.service.SchedulingService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/scheduling")
+@Api(tags = "Endpoints for Scheduling")
+public class SchedulingController {
+
+    private final SchedulingService schedulingService;
+
+    @AdminAccessControl
+    @GetMapping("/interview-time")
+    @ApiOperation("Get all users with their free time for Form")
+    public List<UserDTO> getPossibleInterviewTime(@RequestParam("formId") UUID formId) {
+        return schedulingService.getAdminTimeForForm(formId);
```

Comment:
Question: Does UI know about form status? why backend should decide what users should be returned?
For me , it's logical such flow:
when admin tries to assign user for interview UI should check based on form status and get Users with appropriate role and free time slots.
Then when admin selects appropriate user (admin or tech specialist) and try to assign him to form backend check is it possible to assign or not.

cc: @SergeyKoval @viktarkolbik

Reply:
UI know about status. Do you want to do these checks on the frontend?